Makefile

```diff
@@ -269,14 +270,13 @@ pytest_all:
 	@for i in $$(find projects -name "test_*.py"); 													\
 	do																								\
 		python -m unittest discover -t $$(dirname $$i) -s $$(dirname $$i) -p $$(basename $$i);		\
-	done			
+	done		
 
 .PHONY: install_requirements
 install_requirements:
-	@for i in $$(find projects -name "requirements.txt"); 		\
-	do															\
-		pip install -r $$i;										\
-	done								
+	@rm -rf .venv
+	@virtualenv .venv
+	@pip install -r requirements.txt									
```

Comment:
Are you sure this is using the venv pip? (i.e. should it be `.venv/bin/pip`?) Or maybe should you automatically do `source .venv/bin/activate`? (though that might not work if every line is a separate shell)